ingest/indexer/service/blockprocessor/pair_publisher.go

```diff
@@ -39,7 +40,7 @@ func NewPairPublisher(client domain.Publisher, poolManagerKeeper domain.PoolMana
 // Returns error if at least one of the pairs failed to be published.
 // Nil otherwise.
 // TODO: unit test
-func (p PairPublisher) PublishPoolPairs(ctx sdk.Context, pools []poolmanagertypes.PoolI) error {
+func (p PairPublisher) PublishPoolPairs(ctx sdk.Context, pools []poolmanagertypes.PoolI, createdPoolIDs map[uint64]commondomain.PoolCreation) error {
```

Comment:
I think that we should unit test this method but it can be done in a separate PR - a task is sufficient for now.

Reply:
Good idea! Ticket created:
https://linear.app/osmosis/issue/IND-127/add-test-case-for-pair-publisherpublishpoolpairs